### DISCUSSION TITLE
FIDI detecting duplicates of deleted transactions

### DISCUSSION THREAD

**Discussion by creeg2** [firefly-iii/firefly-iii#6353] [Firefly III] [answered]
I'm testing out importing transactions with FIDI.  I import a csv file, check the results, then delete the transactions.  When I try to reimport the csv file again, I get error messages that lines are duplicate.  I presently do not have any transactions showing in my database (using the [/all](transactions/withdrawal/all) filter on withdrawals or deposits).

Is there a way to resolve this?  Thanks.

![image](https://user-images.githubusercontent.com/27752185/185530012-c34beeba-0f44-48dc-9719-c8796f737beb.png)

**Comment by JC5** [comment, marked as answer]
https://docs.firefly-iii.org/data-importer/help/faq/#why-cant-i-import-duplicate-transactions

**Comment by timaschew** [comment]
~~There are two options to ignore duplicates, on in the webinterface of the importer, but that one doesn't help. I think that one is for duplicates within a single CSV file.~~

~~You need to set the environment variable: `IGNORE_DUPLICATE_ERRORS=true` and if you're using docker-compose, you need to run `docker-compose down` before you start the containers again.~~

Setting `IGNORE_DUPLICATE_ERRORS` to true avoids just the errors in the UI, but you still see them in the server logs.

**Reply by timaschew** [reply]
Actually that doesn't help, I still can't import it, is there any other variable which I need to set.

@JC5

**Reply by JC5** [reply]
In the configuration file, set the detection to "none", or select the same option in the UI.

**Reply by timaschew** [reply]
Ah, my mistake. I didn't read carefully and though that the checkbox (for duplicate within the same file) is turning off the whole detection, also on existing db entries.
